library/Fission/Email/Class.hs

```diff
@@ -0,0 +1,11 @@
+module Fission.Email.Class (MonadEmail (..)) where
+
+import           Fission.Prelude
+import           Servant.Client
+
+import           Fission.Email.Recipient.Types
+import           Fission.Email.SendInBlue.Types as SIB
+import           Fission.Challenge.Types
+
+class Monad m => MonadEmail m where
+  sendVerifyEmail :: Recipient -> Challenge -> m (Either ClientError SIB.Response)
```

Comment:
Nitpick, feel free to ignore: `sendVerificationEmail` or `sendEmailVerification`